packages/safe-ds-eda/types/state.d.ts

```diff
@@ -77,26 +77,25 @@ export interface Profiling {
 
 export interface ProfilingDetailBase {
     type: 'numerical' | 'image' | 'name';
```

Comment:
If you use discriminating unions (here the type `ProfilingDetails`), there's no reason to include the type field in a parent interface.